server/server.js

```diff
@@ -154,20 +156,20 @@ app.use((req, res, next) => {
 
```

Comment:
Is the `app.use` middleware directly above this still needed?

Reply:
Looking at it again, I think we do. Because it'll log people out when the cookie expires. So not a problem.